installer/resources/lambda_submit/function.py

```diff
@@ -126,7 +175,7 @@ class DataShipperCloudWatchEventTarget(CloudWatchEventTargetResource):
     target_id = 'DataShipperTarger'  # Unique identifier
     target_input = json.dumps({
         'jobName': "aws-redshift-es-data-shipper",
-        'jobUuid': "data-shipper",
+        'jobUuid': "data-shipper-jar-with-dependencies",
```

Comment:
Why do we need to name the jar -jar-with-dependencies. It should just be data-shipper.

Reply:
I need to recheck this